src/grt/test/report_wire_length3.ok

```diff
@@ -0,0 +1,26 @@
+[INFO ODB-0222] Reading LEF file: sky130hs/sky130hs.tlef
+[INFO ODB-0223]     Created 13 technology layers
+[INFO ODB-0224]     Created 25 technology vias
+[INFO ODB-0226] Finished LEF file:  sky130hs/sky130hs.tlef
+[INFO ODB-0222] Reading LEF file: sky130hs/sky130hs_std_cell.lef
+[INFO ODB-0225]     Created 390 library cells
+[INFO ODB-0226] Finished LEF file:  sky130hs/sky130hs_std_cell.lef
+[INFO ODB-0127] Reading DEF file: report_wire_length1.def
+[INFO ODB-0128] Design: gcd
+[INFO ODB-0130]     Created 1 pins.
+[INFO ODB-0131]     Created 170 components and 1258 component-terminals.
+[INFO ODB-0133]     Created 15 nets and 72 connections.
+[INFO ODB-0134] Finished DEF file: report_wire_length1.def
+[INFO GRT-0237] Net clk global route wire length: 201.60um
+	Layer met3 : 144.00um
+	Layer met4 : 57.60um
+[INFO GRT-0240] Net clk detailed route wire length: 184.95um
+	Layer met1 :  0.21um
+	Layer met2 :  0.18um
+	Layer met3 : 134.97um
+	Layer met4 : 49.58um
```

Comment:
I'm concerned that people will still parse the log file to get out this data.  Perhaps we should have a -file option to dump in csv for analysis as that is the main use case.

Reply:
I was going to suggest the same -- the use of -file option For these reports it will be nice if can we chose to print out either in text format (with some column based formatting)  or csv format

Specifically for this report we can format  one line per net with the following possible format
<netname> , total_wire_length, num of pins,  num of vias, layer1 WL, layer 2 WL, ..... , layer N WL

The first two columns is for the default report and the remaining is for -verbose option

Reply:
@ravi-varadarajan Could you check the file `OpenROAD/src/grt/test/report_wire_length4.rptok`?

Reply:
I like to include a comment line at the start of the file giving the format as "clk 201.6 2 0 " is non-obvious as to its meaning.